src/lib.rs

```diff
@@ -967,15 +1009,13 @@ impl Build {
                     cmd.args.push(format!("-O{}", opt_level).into());
                 }
 
-                if !nvcc {
-                    cmd.args.push("-ffunction-sections".into());
-                    cmd.args.push("-fdata-sections".into());
-                    if self.pic.unwrap_or(!target.contains("windows-gnu")) {
-                        cmd.args.push("-fPIC".into());
-                    }
-                } else if self.pic.unwrap_or(false) {
-                    cmd.args.push("-Xcompiler".into());
-                    cmd.args.push("\'-fPIC\'".into());
+                cmd.nvcc_wrap_arg(self.cuda);
+                cmd.args.push("-ffunction-sections".into());
+                cmd.nvcc_wrap_arg(self.cuda);
+                cmd.args.push("-fdata-sections".into());
```

Comment:
Could this idiom perhaps get wrapped up into a method? Something which internally pairs these two invocations together?

Reply:
Hmm, originally I had a `Tool::wrapped_arg` function that did just that, but I thought this sort of thing might not be exclusive to CUDA, i.e. it might be possible there are other C-like extensions where we want to prefix a subset of possible args but leave the rest untouched, and the subset will invariably differ between these extensions. For example it seems the "-Xcompiler" convention originates at least from libtool. I don't know, what's your take on this rationale?